crates/bevy_ecs/src/schedule/executor/mod.rs

```diff
@@ -108,7 +108,10 @@ impl SystemSchedule {
 }
 
 /// See [`ApplyDeferred`].
-#[deprecated = "Use `ApplyDeferred` instead. This was previously a function but is now a marker struct System."]
+#[deprecated(
```

Comment:
Why not remove this?

Reply:
It was just added like a week ago, just realized I need to set the since to `0.16.0`